pkg/os/volume/api.go

```diff
@@ -170,18 +170,22 @@ func (VolumeAPI) ResizeVolume(volumeID string, size int64) error {
 	} else {
 		finalSize = size
 	}
+	klog.V(2).Infof("finalSize for resize of the volume: %s is %d ", volumeID, finalSize)
```

Comment:
V(2) might be too verbose, we don't use `-v` in OSS though

Reply:
fixed. `used klog.V(4)`.